src/tests/ftest/util/general_utils.py

```diff
@@ -642,7 +642,8 @@ def convert_list(value, separator=","):
     return separator.join([str(item) for item in value])
 
 
-def stop_processes(hosts, pattern, verbose=True, timeout=60, added_filter=None):
+def stop_processes(hosts, pattern, verbose=True, timeout=60, added_filter=None,
+                   send_sigusr2=False):
```

Comment:
Could this be more generic, like:

```suggestion
                   dump_ult_stacks=False):
```